.github/workflows/load-tests-job-configs/java-cogbk/java_CoGBK_Dataflow_Batch_100b_Multiple_Keys.txt

```diff
@@ -0,0 +1,28 @@
+#  Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I think grouping files in the `/java-cogbk` directory is redundant. Let's leave these files in the `load-tests-job-configs` directory